### ISSUE TITLE
Graphite service doesn't work

### ISSUE THREAD

**Issue by podshumok** [influxdata/kapacitor#760]
I'm trying to send TCP graphite traffic directly to kapacitor to avoid UDP influx subscriptions.
I copied `[[graphite]]` section from influx config file, changed port and restarted kapacitor.
Service listens to the port, looks like parsing works too (I see parsing errors, when `netcat`ing garbage to the port), but data are not available in the StreamNode

**Comment by nathanielc** [comment]
Can you share your config and the output of `kapacitor stats ingress`?

Also which version are you using? The beta versions have support for the HTTP based subscriptions so you can avoid the UDP based subscriptions.

**Comment by podshumok** [comment]
I'm using
Kapacitor 0.13.1 (git: HEAD ae7dde3aa467c46c06a5a2058a80c6125203f599)

I'll try to provide mwe later

**Comment by toni-moreno** [comment]
Related to this issue.

@podshumok , @nathanielc Could you report me if [[graphite]] section is already working on Kapacitor?  

 I also would like to know if there is a way to query for data from graphite (only needed related to the tickscript measurement)  rather than duplicate  all my graphite metrics to send also to the kapacitor daemon ? sending

**Comment by nathanielc** [comment]
@toni-moreno No, you cannot query graphite from TICKscripts.

The graphite service is working, if you are having a specific issue please let us know the details.

**Comment by podshumok** [comment]
I finally have found some time to investigate and replicate the problem.
The problem is more or less related to default config file: 
`default-retention-policy ` is set to empty string by default, and I hadn't specified retention policy in `[[graphite]]` section.
Same configuration with influx will put graphite data to retention policy named `default`, so I had defined `TICK` as
`FILE="cpu.tick" sh -c 'kapacitor define $FILE -tick $FILE -dbrp graphite.default --type=stream'`
Now I have changed `default-retention-policy ` from empty string to `default` and looks like everything is working.
